samples/snippets/src/main/java/com/example/spanner/ObservabilityLab.java

```diff
@@ -244,6 +309,13 @@ private void executeTasksInParallel(Runnable task, int numProcessors) {
     }
   }
 
+  /**
+   * Method to execute the transaction locking or non locking method.
+   *
+   * @param dbClient DatabaseClient
+   * @param isLocking value to decide which method to call. True means locking method will be
+   *     called, false means the otherwise.
+   */
```

Comment:
nit: would be great to add these comments into lab 1 code as well, if you get the chance.

Reply:
Sure.